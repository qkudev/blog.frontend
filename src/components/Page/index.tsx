import * as React from 'react'
import './style'

interface Props {
  className?: string
  children?: React.ReactNode
}

function Page({ className, children }: Props) {
  return <div className={`page ${className || ''}`}>{children}</div>
}

export default Page
