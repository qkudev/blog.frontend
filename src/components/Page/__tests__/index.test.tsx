import * as React from 'react'
import renderer from 'react-test-renderer'
import Page from '../index'

describe('Page', function() {
  test('Matches with snapshot', function() {
    const page = renderer.create(<Page />)
    let tree = page.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
