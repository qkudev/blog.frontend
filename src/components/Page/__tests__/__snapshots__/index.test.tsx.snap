// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page Matches with snapshot 1`] = `
<div
  className="page "
/>
`;
